catalog/transformer/catalog-transformer-streaming-api/src/main/java/org/codice/ddf/transformer/xml/streaming/AbstractSaxEventHandler.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.transformer.xml.streaming;
+
+import org.xml.sax.Locator;
+import org.xml.sax.SAXException;
+
+public abstract class AbstractSaxEventHandler implements SaxEventHandler {
+
+    @Override
+    public void setDocumentLocator(Locator locator) {
+
+    }
+
+    @Override
+    public void startDocument() throws SAXException {
+
+    }
+
+    @Override
+    public void endDocument() throws SAXException {
+
+    }
+
+    @Override
+    public void startPrefixMapping(String prefix, String uri) throws SAXException {
+
+    }
+
+    @Override
+    public void endPrefixMapping(String prefix) throws SAXException {
+
+    }
+
+    @Override
+    public void endElement(String uri, String localName, String qName) throws SAXException {
+
```

Comment:
I'll remove the `endElement` and `characters` implementations, but `startPrefixMapping` doesn't look like it's needed as much.